agents/ocf/HealthIOWait.in

```diff
@@ -109,16 +118,16 @@ agent_monitor() {
           # echo $OCF_RESKEY_yellow_limit
           if [ $WAIT -gt ${OCF_RESKEY_red_limit} ] ; then
             # echo "System state RED!"
-            attrd_updater -n "#health-iowait" -U "red" -d "5s"
+            attrd_updater -n "#health-iowait" -U "red" -d ${OCF_RESKEY_dampening}
```

Comment:
I missed this for the first one, but it would be worthwhile to quote the variable in case the user uses a space by mistake. The command might fail but it should give a good error message rather than behave unpredictably.

Reply:
Yes, this writing does have the problem you mentioned

Reply:
> I missed this for the first one, but it would be worthwhile to quote the variable in case the user uses a space by mistake. The command might fail but it should give a good error message rather than behave unpredictably.

> if [ -z "${OCF_RESKEY_dampening}" ] || [[  #${OCF_RESKEY_dampening} =~ ^[[:space:]]+$ ]]; then

If the user enters multiple Spaces, the default value is entered.

Reply:
I meant like `-d "${OCF_RESKEY_dampening}"`, in case the user sets something like dampening="5000 ms". Without quotes, `-d 5000 ms` will be interpreted as `-d 5000` (seconds) plus an extraneous `ms` argument that would either be ignored or cause a usage error. With quotes, `-d "5000 ms"` might work or might give an error message, but at least the error message would be point to the right problem.

Reply:
Oh, I see. I agree with you. It's very thoughtful. I will put the dampening parameter in quotes.